custodia/server/__init__.py

```diff
@@ -100,7 +101,14 @@ def _create_plugin(parser, section, menu):
 
 
 def parse_config(args):
-    parser = ConfigParser(interpolation=ExtendedInterpolation())
+    defaults = {
+        'fqdn': socket.getfqdn(),
```

Comment:
for the record socket.getfqdn() calls socket.gethostname(). Both variants are equivalent and always use gethostbyaddr.

Let's use gethostname() and include a comment why we use gethostname rather than getfqdn.